bdk-ffi/src/tx_builder.rs

```diff
@@ -18,6 +17,8 @@ use std::convert::{TryFrom, TryInto};
 use std::str::FromStr;
 use std::sync::Arc;
 
+type ChangeSpendPolicy = bdk_wallet::ChangeSpendPolicy;
```

Comment:
more out of curiosity (and not having as many reps as you on this) than me thinking its wrong or anything, what made you have to do this (function, aesthetic, etc)?